apps/i18n/eval/en_us.json

```diff
@@ -6,6 +6,7 @@
   "ellipseBlockTitle": "ellipse (width, height, style, color)",
   "emptyFunctionalBlock": "You have a block with an unfilled input.",
   "extraTopBlocks": "You have unattached blocks. Did you mean to attach these to the \"display\" block?",
+  "infiniteRecursionError": "Infinite recursion detected in your code.",
```

Comment:
Should we give students a little more info about what's wrong?  This error message sounds a little cryptic :)